ExportScripts/export_standalone_monolithic_windows.py

```diff
@@ -0,0 +1,285 @@
+
+
+    game_asset_list_path = args.project_path /'AssetBundling'/'AssetLists'/'game_pc.assetlist'
+    seed_folder_path = args.project_path/'AssetBundling'/'SeedLists'
+
+    game_asset_list_command = [asset_bundler_batch_path, 'assetLists', '--assetListFile', game_asset_list_path, 
+                    '--seedListFile', seed_folder_path  / 'BasePopcornFxSeedList.seed',
+                    '--seedListFile', seed_folder_path  / 'GameSeedList.seed']
+
+    if args.config == 'profile':
+        game_asset_list_command += ['--seedListFile', seed_folder_path / 'ProfileOnlySeedList.seed']
+
+    game_asset_list_command += ['--seedListFile', seed_folder_path / 'VFXSeedList.seed', '--project-path', args.project_path, '--allowOverwrites']
+
+    process_command(game_asset_list_command, cwd=args.engine_path)
+
+    engine_bundle_path = args.project_path / 'AssetBundling' / 'Bundles' / 'engine_pc.pak'
+    process_command([asset_bundler_batch_path, 'bundles', '--assetListFile', engine_asset_list_path, '--outputBundlePath', engine_bundle_path, '--project-path', args.project_path, '--allowOverwrites'], cwd=args.engine_path)
+
+    # This is to prevent any accidental file locking mechanism from failing subsequent bundling operations
+    time.sleep(1)
+
+    game_bundle_path   = args.project_path / 'AssetBundling' / 'Bundles' / 'game_pc.pak'
+    process_command([asset_bundler_batch_path, 'bundles', '--assetListFile', game_asset_list_path, '--outputBundlePath', game_bundle_path, '--project-path', args.project_path, '--allowOverwrites'], cwd=args.engine_path)
+
+
+    #Create Launcher Zip File
+    os.makedirs( args.output_path / 'Cache' / 'pc', exist_ok=True)
+    os.makedirs( args.output_path / 'Gems' / 'AWSCore', exist_ok=True)
+
+    process_command(['xcopy', '/Y', pathlib.PurePath(args.project_path / 'AssetBundling' / 'Bundles' / '*.pak'),
+                            args.output_path / 'Cache' / 'pc'], cwd=args.engine_path)
+    process_command(['xcopy', '/Y', pathlib.PurePath(mono_build_path / 'bin' / 'profile' / '*.*'),
+                            args.output_path], cwd=args.engine_path)
+    process_command(['xcopy', '/Y', pathlib.PurePath(mono_build_path  / 'bin' / 'profile' / 'Gems' / 'AWSCore' / '*.*'),
+                            args.output_path / 'Gems' / 'AWSCore'], cwd=args.engine_path)
+    process_command(['xcopy', '/Y', pathlib.PurePath(args.project_path / 'launch_*.*'), args.output_path], cwd=args.engine_path)
+
+    if args.zip_output:
+        import shutil
+        archive_name = args.output_path
+        print("Zipping output directory (this may take a while)...")
+        shutil.make_archive(args.output_path, 'zip', root_dir = args.output_path)
+
+    print(f"Exporting project is complete! Release Directory can be found at {args.output_path}")
+
+    process_command(['explorer', args.output_path])
```

Comment:
I recommend having the `quiet` option skip such an operation.

Reply:
Yes the intention was to replicate Windows unzip by showing the result of the build process. But based on the instructions for PackedAssetBuilds, that's not as helpful as I originally thought, because you have to run the executables via command line (if you can double click the exe file, then it's still helpful I think). Maybe a new terminal window at the output location would be easier to work with.